### PR TITLE
Fix/fish groups

### PR DESCRIPTION
**Pull request by meganrm**
The wrong fish target was being grayed out on the default page: [staging.cfe.allencell.org](http://staging.cfe.allencell.org/?cellSelectedFor3D=F0219C05&colorBy=cell-age&dataset=cellsystems_fish_v2021.1&plotByOnX=cos&plotByOnY=cell-area&selectedPoint%5B0%5D=F0219C05)
I already ran the script so the correct FISH target should be on by default, but there is another issue with the `HPRT1-COL2A1` being grayed out that I have a fix for on the front end. 

@tanyasg can you check your own scripts to make sure this change makes sense? I don't want us to update this data in the future and then re-introduce this bug. 
Basically the change is 
`0` used to map to `MYH6-MYH7` and `1` used to map to `HPRT1-COL2A1`
And now: 
`0` maps to `HPRT1-COL2A1` and `1` maps to `MYH6-MYH7`

I made this switch based on the cell counts for each target. All the other ones seemed correct.

### CODE REVIEW

**Review by schoinh — APPROVED** [review]

**Review by tanyasg — APPROVED** [review]
@meganrm this change makes sense and won't affect anything on my end. I accidentally swapped the two when I was adding the gene pairs to the feature json.
